tests/constants.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+"""Constants for tests."""
+
+from pathlib import Path
+
+HERE = Path(__file__).parent.absolute()
```

Comment:
should check to make sure that the tests are being checked by the flake8 script - there are a lot of parts here that should be caught